photolog/views.py

```diff
@@ -25,8 +26,16 @@ def decorate_view(*args, **kwargs):
 @photolog.route('/', methods=['GET'])
 @photolog.route('/<int:page>', methods=['GET'])
 def index(page=1):
+    pagination = {
+        'current_page': page,
+        'per_page': ITEMS_PER_PAGE,
+        'total_albums': Albums.query.count()
+    }
+    pagination['total_pages'] = int(pagination['total_albums'] / pagination[
+        'per_page']) + 1
```

Comment:
Taken care of
![surprise_buttsecks](https://cloud.githubusercontent.com/assets/2191211/21741288/1d17205e-d4d1-11e6-8313-71babe9cee30.gif)